morphocut/str.py

```diff
@@ -32,11 +56,29 @@ class ParseWarning(UserWarning):
 @ReturnOutputs
 @Output("meta")
 class Parse(Node):
-    """Parse information from a path.
+    """Parse information from a string. The class can be used to simply parse a string
+    Or to search a string in some pattern.
```

Comment:
Singe line summary, blank line, detailed description. Second sentence is a rephrasing of the first, remove. Remove reference to search, it does not do that.

All in all: Just use this quote from the parse documentation: ``Parse strings using a specification based on the  :ref:`Python Format String Syntax <python:formatstrings>`.``